charts/substra-backend/templates/deployment-worker.yaml

```diff
@@ -112,8 +112,10 @@ spec:
 {{ toYaml . | indent 12 }}
           {{- end }}
           volumeMounts:
+            {{- if .Values.backend.dockerSocket.enabled }}
             - name: dockersocket
               mountPath: /var/run/docker.sock
```

Comment:
seems that it should be updated also here {{ .Values.backend.dockerSocket.hostPath }}

Reply:
This variable is here to know where the docker socket is on the host.
As docker socket on pods should be in /var/run/docker.sock there is no issue here.